services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/logging/DisclosureLogService.kt

```diff
@@ -151,7 +151,8 @@ class DisclosureLogService(private val auditLogService: AuditLogService) {
                     .filter { it.type == CustomerType.PERSON }
         }
 
-    fun extractYhteystiedot(hanke: Hanke) = hanke.omistajat + hanke.arvioijat + hanke.toteuttajat
+    fun extractYhteystiedot(hanke: Hanke) =
+        hanke.omistajat + hanke.rakennuttajat + hanke.toteuttajat + hanke.muut
```

Comment:
FFTI: This pattern of combining the different kinds of yhteystiedot is repeated a couple of times. It could be beneficial to move this function to Hanke and use that instead.

Reply:
Done.